src/Draggable/Plugins/Focusable/Focusable.js

```diff
@@ -51,6 +51,11 @@ export default class Focusable extends AbstractPlugin {
    */
   detach() {
     this.draggable.off('draggable:initialize', this[onInitialize]).off('draggable:destroy', this[onDestroy]);
+
+    // Remove modified elements when detach
+    requestAnimationFrame(() => {
+      this.getElements().forEach((element) => stripElement(element));
+    });
```

Comment:
How about change it to 
```
this[onDestroy]();
```

Reply:
Ah, I see, it was duplicated. It ok, wait a minute 😄

Reply:
@zjffun , I changed it 😄